ntbs-integration-tests/LabResultsPage/LabResultsPageTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using ntbs_integration_tests.Helpers;
+using ntbs_integration_tests.TestServices;
+using ntbs_service;
+using ntbs_service.Services;
+using Xunit;
+
+namespace ntbs_integration_tests.LabResultsPage
+{
+    public class LabResultsPageTests : TestRunnerBase
```

Comment:
Might want at least one test to do CanNotViewSpecimensAccordingToPermissions to check the condition for seeing specimens is working correctly